app/src/main/res/layout/document__fragment__edit.xml

```diff
@@ -30,8 +30,9 @@
         <net.gsantner.markor.frontend.textview.HighlightingEditor
             android:id="@+id/document__fragment__edit__highlighting_editor"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_height="wrap_content"
             android:background="@color/background"
+            android:minHeight="1dp"
```

Comment:
I have no idea why this 1dp is necessary. I suspect that ever 1px would work. But this + wrap_content solves the padding issue